Content.Server/GameObjects/Components/Damage/DestructibleComponent.cs

```diff
@@ -53,9 +54,15 @@ void IOnDamageBehavior.OnDamageThresholdPassed(object obj, DamageThresholdPassed
             if (e.Passed && e.DamageThreshold == Threshold && destroyed == false)
             {
                 destroyed = true;
-                var wreck = Owner.EntityManager.SpawnEntity(spawnOnDestroy);
-                wreck.Transform.GridPosition = Owner.Transform.GridPosition;
-                Owner.EntityManager.DeleteEntity(Owner);
+                var coord = Owner.Transform.GridPosition;
+                var entMgr = Owner.EntityManager;
+                Owner.Delete();
+                if (spawnOnDestroy.Length > 0) {
```

Comment:
You might want to check this against trimmed spawnOnDestroy so strings with only whitespace don't trigger this.

Reply:
The correct way to check this is `string.IsNullOrWhiteSpace`, fyi.